src/smitaVetal/LengthCharReturn.java

```diff
@@ -0,0 +1,42 @@
+/*// 1. write a method to return length of the given name
+//int getNameLength(String name)
+//2. return middle character from given name.
+//Hint : char getMiddleChar(String name)
+//input : Maulik
+//output : Middle character in name Maulik is u
+//input : Technocredits
+//output : Middle character in name Technocredits is c*/
+
+package smitaVetal;
+
+public class LengthCharReturn {
+
+	char getMiddleChar(String str) {
+		int length = str.length();
```

Comment:
as you created method which calculate length of string then why we are direct calling string method?